egs2/TEMPLATE/asr1/path.sh

```diff
@@ -19,4 +19,4 @@ export PYTHONIOENCODING=UTF-8
 export NCCL_SOCKET_IFNAME="^lo,docker,virbr,vmnet,vboxnet"
 
 # NOTE(kamo): Source at the last to overwrite the setting
-. local/path.sh
+#. local/path.sh
```

Comment:
```suggestion
. local/path.sh
```
Please not edit this.